src/wallet/rpcwallet.cpp

```diff
@@ -30,6 +30,11 @@ using namespace std;
 int64_t nWalletUnlockTime;
 static CCriticalSection cs_nWalletUnlockTime;
 
+CWallet *GetWalletForJSONRPCRequest(const JSONRPCRequest& request)
```

Comment:
Can you document this function somewhere (also probably move its definition to src/wallet/rpcwallet.h) - a few places in src/wallet assume it always returns non-NULL but this is not documented (I havent looked at the actual multi-wallet PR, but does this function then throw an RPC exception if you ask for a wallet that isnt loaded, or does it return NULL then)?